src/installer/corehost/cli/hostpolicy/coreclr.h

```diff
@@ -66,6 +66,7 @@ enum class common_property
     AppNIPaths,
     RuntimeIdentifier,
     BundleProbe,
+    PInvokeOverride,
```

Comment:
This seems bad. We just broke compatibility by inserting this value into the enum list Unless this is entirely an implementation detail. I would suggest we hard code these values unless order doesn't impact sxs scenarios.

Reply:
I think it is an implementation detail just for the property bag where this stuff is accumulated. I'll check to be sure.
Might still be a good idea to add the new value at the end though.

Reply:
This is purely an implementation detail - there's a matching string table, and that's the only place which needs to match the order of the enum. Everything else goes by the num. I must admit I don't know why it's done this way... we could as well have a class of static strings and use those as "enum" as well as the value.

Reply:
Considering this is an implementation detail, I will just make the new value last in the enum.